docs/section11/Rule_11-10.md

```diff
@@ -0,0 +1,285 @@
+    - if the tank_type is CONSUMER_STORAGE or COMMERCIAL_STORAGE, set the type to STORAGE: `if tank_type in ["CONSUMER_STORAGE", "COMMERCIAL_STORAGE"]: type = "STORAGE"`
+    - get the fuel type of the equipment: `fuel_type = shw_eq.heater_fuel_type`
+    - set the type based on tank_type and fuel type: `if fuel_type == "ELECTRICITY":`
+      - `if tank_type == "INSTANTANEOUS": type = "ELECTRIC_RESISTANCE_INSTANTANEOUS"`
+      - `elsif tank_type == "STORAGE": type = "ELECTRIC_RESISTANCE_STORAGE"`
+    - `elsif fule_type == "NATURAL_GAS":`
+      - `if tank_type == "INSTANTANEOUS": type = "GAS_INSTANTANEOUS"`
+      - `elsif tank_type == "STORAGE": type = "GAS_STORAGE"`
+    - `elsif fule_type == "FUEL_OIL":`
+      - `if tank_type == "INSTANTANEOUS": type = "OIL_INSTANTANEOUS"`
+      - `elsif tank_type == "STORAGE": type = "OIL_STORAGE"`
+    - if the draw_pattern is given, get the draw pattern: `if shw_eq.draw_pattern != nil: draw_pattern = shw_eq.draw_pattern`
+    - otherwise, determine the draw pattern based off the first_hour_rating of the shw_eq: `else:`
+      - First get the first hour rating (gallons): `first_hour_rating = shw_eq.first_hour_rating`
+      - if the first_hour_rating is less than 18 gallons, draw_pattern is "VERY SMALL": `if first_hour_rating < 18: draw_pattern = "VERY_SMALL"`
+      - otherwise if the first_hour_rating is less than 51 gallons, draw pattern is LOW: `elsif first_hour_rating < 51: draw_pattern = "LOW"`
+      - otherwise if the first_hour_rating is less than 75 gallons, draw pattern is MEDIUM: `elsif first_hour_rating < 75: draw_pattern = "MEDIUM"`
+      - otherwise draw pattern is HIGH: `else: draw_pattern = "HIGH"`
+    - get the capacity of the SHW system (make sure to convert to btu/hr): `input_capacity = swh_eq.input_power`
+    - get the volume of the tank (make sure to convert to gallons): `storage_volume = swh_eq.distribution_system.tank.storage_capacity`
+    - calculate the capacity_per_volume based on the input_capacity and the storage_volume: `capacity_per_volume = shw_eq.rated_capacity / storage_volume`
+    - get the standby loss (see note 1 for calculation source): `standby_loss = swh_eq.standby_loss_fraction * 8.25 * volume * 70`
+    - what follows is logic that implements Table 7.8 - this table has multiple criteria, so I think it's simpler to implement with if / else logic than a table lookup
+    - if the type is OTHER, set rule_note to "Water heater type is not a recognized type": `if type == "OTHER": rule_note = "Water heater type is not a recognized type"`
+    - otherwise, continue with rule logic: `else:`
+      - create a boolean has_fault and set it to false: `has_fault = false`
+      - if the water heater is electric resistance, or storage and the capacity is <= 12kW (40945 btu/hr), look at Appendix F: `if type in ["ELECTRIC_RESISTANCE_INSTANTANEOUS", "ELECTRIC_RESISTANCE_STORAGE"] && input_capacity <= 40945.7:`
```

Comment:
Why are we checking instantaneous efficiency since the baseline is always a storage water heater? Maybe there is something below that will clarify.

Reply:
removed all references to instantaneous and return UNDETERMINED if type is instantaneous